persistence-cassandra-3.11/src/main/java/io/stargate/db/cassandra/impl/SchemaConverter.java

```diff
@@ -125,6 +129,20 @@ protected boolean isCustom(IndexMetadata index) {
     return index.kind == IndexMetadata.Kind.CUSTOM;
   }
 
+  @Override
+  protected String indexClass(IndexMetadata index) {
+    return index.options.get("class_name");
+  }
+
+  @Override
+  protected Map<String, String> indexOptions(IndexMetadata index) {
+    final Set<String> excludeOptions =
+        ImmutableSet.of(IndexTarget.CUSTOM_INDEX_OPTION_NAME, IndexTarget.TARGET_OPTION_NAME);
```

Comment:
Can you extract this as a constant?

Reply:
I have extract as a constant and pull it up to the (abstract) parent class. Is this second refactoring worth doing?